From 570cefb02b4f149c4263677e1a3eea216ee0ec8c Mon Sep 17 00:00:00 2001
From: Luca Barbato <lu_zero@gentoo.org>
Date: Wed, 17 Sep 2014 19:51:40 +0200
Subject: [PATCH 05/10] hevc: Initialize mergecand_list to 0

Unbreak cf6090dc6252f2b276aa4133e3d73a89f4c6046c.

CC: libav-stable@libav.org
Sample-Id: hevc-conformance-LTRPSPS_A_Qualcomm_1
---
 libavcodec/hevc_mvs.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libavcodec/hevc_mvs.c b/libavcodec/hevc_mvs.c
index cc5a16c..a611b76 100644
--- a/libavcodec/hevc_mvs.c
+++ b/libavcodec/hevc_mvs.c
@@ -556,6 +556,8 @@ void ff_hevc_luma_mv_merge_mode(HEVCContext *s, int x0, int y0, int nPbW,
     int nPbH2 = nPbH;
     HEVCLocalContext *lc = &s->HEVClc;
 
+    memset(mergecand_list, 0, MRG_MAX_NUM_CANDS * sizeof(*mergecand_list));
+
     if (s->pps->log2_parallel_merge_level > 2 && nCS == 8) {
         singleMCLFlag = 1;
         x0            = lc->cu.x;
-- 
2.1.0

